libfwupdplugin/fu-firmware.h

```diff
@@ -48,12 +48,16 @@ struct _FuFirmwareClass
  * @FU_FIRMWARE_FLAG_NONE:			No flags set
  * @FU_FIRMWARE_FLAG_DEDUPE_ID:			Dedupe imges by ID
  * @FU_FIRMWARE_FLAG_DEDUPE_IDX:		Dedupe imges by IDX
+ * @FU_FIRMWARE_FLAG_HAS_CHECKSUM:		Has a CRC or checksum to test internal consistency
+ * @FU_FIRMWARE_FLAG_HAS_VID_PID:		Has a vendor or product check in the firmware
```

Comment:
I don't think you can prove it's a product / vendor **check**.  You can only prove that the firmware image contains an identifier of some sort.  Whether or not it's validated, that's a different story.

Reply:
Well, I figured `HAS` kinda implies that the information is there, rather than we verify it.

Reply:
But the comment says `check`

Reply:
Ha, sorry! What about `Has a vendor or product ID in the firmware`

Reply:
+1

Reply:
Fixed, thanks.